asv_bench/benchmarks/benchmarks.py

```diff
@@ -127,12 +128,56 @@ def time_join(self, shapes, how, sort):
         execute(self.df1.join(self.df2, how=how, lsuffix="left_", sort=sort))
 
 
+class TimeJoinStringIndex:
+    param_names = ["shapes", "sort"]
```

Comment:
We likely want to benchmark `left` and `inner` values for `how` parameter, don't we?

Reply:
We already have benchmarks for the parameter. I think we don't need another benchmark for this as it seems like a duplication to me.